README.md

```diff
@@ -1,24 +1,153 @@
-# README
+# Book A Swimming Class Backend
```

Comment:
_note: Since there is no open PR in the frontend repo for me to review, I'll include my comments here for that project._

# Frontend review

- [ ]  Please, kindly refactor `App` and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/231926980-7829a229-cce9-440c-8899-cafdc6158bc2.png)

That way, your code will meet the latest standards. 💪 

- [ ] This component is consuming the API non-stop on the homepage:

![image](https://user-images.githubusercontent.com/57421823/231929072-99e91f3f-e20a-4356-8025-84a084201907.png)

![image](https://user-images.githubusercontent.com/57421823/231929089-f1277d32-9ccf-44b6-ad26-d487c3fbac8a.png)

The more time I spend on the "Classes" page, the more requests to the API it does. 

The issue is in the `useEffect` hook call.  Kindly take a look so this bug is fixed and your app works as intended. 👍 

- [ ] When I try to make a reservation, It works on the server, but the frontend won't show the reservations:

![image](https://user-images.githubusercontent.com/57421823/231929726-bf8e552a-25b5-4391-ab7e-bfa553cfb3dd.png)

According to the Rails console, I already have 3 reservations in the database:

![image](https://user-images.githubusercontent.com/57421823/231929809-7f1c8050-e767-43c9-873d-5b41b907f790.png)

Kindly, check your React routes so this bug is fixed, and your app works as intended. 👍 

- [ ] In the mobile view, some containers are overflowing:

![image](https://user-images.githubusercontent.com/57421823/231930428-fec58ebb-1c97-4740-a7f5-33f9600bd0ad.png)

Kindly, take a look at your styling so the containers don't overflow on each other.  That way, your mobile view will be beautiful. 💪